_posts/2017-05-14-canvass.md

```diff
@@ -0,0 +1,46 @@
+---
+layout:     post
+title:      A/B Testing in Javascript with Canvass
+date:       2017-05-14 15:45:00
+summary:    A/B testing is a popular method for testing changes. Here's how we do it with the Canvass library.
+author:     adam_blanchard
+image:      test-tube.png
+category:   Software Engineering
+tags:       javascript, open-source, experimentation, abtesting
+---
+A/b testing is a popular method used to identify new changes or features which increase some metric (often the conversion rate in our industry). It works by creating a variant of the website which is served to only a subset of the users. It's performance is then compared to the current version of the website.
```

Comment:
Probably want to make sure A/B is capitalised everywhere.
"Its" performance, not "It's".

How is the performance compared to the current version?  What is being compared?  The metric is mentioned above, but I think it needs making a little more explicit, even as an introduction to a relatively well-understood subject.